### ISSUE TITLE
[A11y]Tooltip for "Package has Vulnerabilities" icon in version list renders offscreen when screen is narrow

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9424]
Url: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1744522

Environment Details:
App name: NuGet Service
#URL: [https://www.nuget.org](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.nuget.org%2f&data=05%7c01%7cv-nitipi%40microsoft.com%7c9a84307e7aa744a4b24208dafa5ca2fe%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638097573052590837%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=C8Eab3i6ixAmqsOjG4AxNT36z9WZQiXPbjDDkMcS5ME%3D&reserved=0)

OS and Browser Details:
Edition Windows 11 Enterprise
OS build 22621.1105.
Chrome version:  109.0.5414.120 (Official Build) (64-bit)

Pre-Requisites:
Press "Ctrl+Shift+I" to open developer tool.
Now in dock side click on "undock into separate window" so that developer tool will get open in separate window.
Zoom the browser of the screen to be tested to 200%.
Adjust the screen to 320px*256px.

Repro Steps:
Hit the URL: https://www.nuget.org/
Navigate to "Sign in" control and hit enter.
Navigate to "Sign in with Microsoft" control and hit enter.
Search for a key term (say “json”) from search box.
Click on "Newtonsoft.json" control and hit enter.
Navigate to "Version History" control and hit enter.
Apply reflow and observe whether "Version history" tooltip is not visible.

Actual Result:
On applying reflow, Version history tooltip in "Version" tab is not visible.

Expected Result:
On applying reflow, "Version history" tooltip in "Version" tab content should be completely visible.

Notes:
This tooltip is a custom (I believe Bootstrap) control and thus doesn't always follow browser expectation when rendering. As such, I believe it is rendering outside of the viewport when it renders on a narrow screen. Not 100% sure what the best option here is. One option is to have it render on the other side (left instead of right) when screen is narrow).

**Comment by agr** [comment]
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1744542 - almost the same issue.

**Comment by agr** [comment]
Both A11y issues are confirmed and closed.
